scripts/jenkins-yoctobuild-build.sh

```diff
@@ -83,6 +83,17 @@ EOF
          "$PR_STATUS_ENDPOINT"
 }
 
+
+board_support_update() {
+    BOARD=$1
+    STATUS=$2
```

Comment:
In my matrixed Jenkins branch I've started to use `local` to get things a bit more contained. Maybe we can start with that here as well?